aspnetcore/migration/21-to-22.md

```diff
@@ -124,6 +124,10 @@ The following table shows the Docker image tag changes:
 
 Change the `FROM` lines in your *Dockerfile* to use the new image tags in the preceding table's 2.2 column.
 
+## Build manually in Visual Studio when using IIS in-process hosting
+
+Visual Studio's **Auto build on browser request** experience doesn't function with the [IIS in-process hosting model](xref:fundamentals/servers/aspnet-core-module#in-process-hosting-model). You must manually rebuild the project when using in-process hosting. We plan to improve this experience in a future release of Visual Studio.
```

Comment:
```suggestion
Visual Studio's **Auto build on browser request** experience doesn't function with the [IIS in-process hosting model](xref:fundamentals/servers/aspnet-core-module#in-process-hosting-model). You must manually rebuild the project when using in-process hosting. This experience will improve in a future release of Visual Studio.
```

Reply:
I wrote the sentence that way specifically to avoid future tense. I also prefer to say that engineering "plans" to address it.

Reply:
> We're looking to improve that in VS2019.

"looking to" == "plan to"

Reply:
I was just trying to avoid "we".

Reply:
https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/w/we

If "we recommend" is ok, then I think "we plan" is ok, too.

Reply:
We can't make the community member the focus of this line. It's basically has to say that MS is doing something here imo.

Reply:
> A plan is in place to improve this experience in a future release of Visual Studio.

🎤*drop* :smile:

Reply:
*Ah! I got it!* ....

> Improvements to this experience are planned for a future release of Visual Studio.